BackofficeBundle/Manager/RouteDocumentManager.php

```diff
@@ -42,15 +42,16 @@ public function __construct(
     }
 
     /**
-     * @param NodeInterface $node
+     * @param NodeInterface $givenNode
      *
      * @return array
      */
-    public function createForNode(NodeInterface $node)
+    public function createForNode(NodeInterface $givenNode)
```

Comment:
I do this renaiming because the node we are going to use to create the routes will not be the same